relay-server/src/actors/processor.rs

```diff
@@ -1639,6 +1639,20 @@ impl EnvelopeProcessorService {
                     sdk = envelope.meta().client_name().unwrap_or("proprietary"),
                     platform = event.platform.as_str().unwrap_or("other"),
                 );
+
+                let otel_context = event
+                    .contexts
+                    .value()
+                    .and_then(|contexts| contexts.get("otel"))
+                    .and_then(Annotated::value);
+
+                if otel_context.is_some() {
+                    metric!(
```

Comment:
Yes this was my intention, the same as transaction source metrics we are collecting.

This will be the estimate we use to understand the consequences of doing more complicated processing of opentelemetry events in Relay, details of which we will share with the Ingest team very soon.